third_party/llvm-project/include/llvm/ObjectYAML/DWARFYAML.h

```diff
@@ -151,6 +151,7 @@ struct LineTableOpcode {
 };
 
 struct LineTable {
+  uint64_t Position; // XXX BINARYEN: the binary location in .debug_line
```

Comment:
This is set but not used anywhere in this PR?

Reply:
It's used here: https://github.com/WebAssembly/binaryen/pull/3700/files#diff-62256ba16b71cbddf493d990a92cc71e51a3f2ae0a57289ae85ae5d794db3d31R765

We set this during load. During rewrite, we read these values so we know the old position. (We also update it there, although we do not currently read at any later time - I added that to avoid future confusion, seems best to the keep the IR consistent.)